packages/GA4Client/test/src/tests.js

```diff
@@ -935,9 +938,10 @@ describe('Google Analytics 4 Event', function () {
                         promotion_name: 'Winter Sale Banner',
                         creative_name: 'Winter Sale',
                         creative_slot: 'featured_app_2',
+                        send_to: 'testMeasurementId',
                     },
                 ];
-
+                
```

Comment:
Not sure if you're running this through our linters before committing, but let's kill this whitespace.

```suggestion

```